nav2_bt_navigator/behavior_trees/navigate_to_pose_w_replanning_goal_patience_and_recovery.xml

```diff
@@ -0,0 +1,49 @@
+
+<!--
+  This Behavior Tree replans the global path periodically at 1 Hz and it also has
+  recovery actions specific to planning / control as well as general system issues. 
+  This will be continuous if a kinematically valid planner is selected.
+  Additionally it has a feature to wait near the goal to have a new valid plan 
+  if the old plan becomes invalid before trying to drive significantly 
```

Comment:
Its not that its invalid, its that it just asked the robot to drive around the "long way" vs waiting to see if the obstacle at the goal pose moves